conans/cli/output.py

```diff
@@ -1,11 +1,23 @@
-import logging
 import sys
 
 from colorama import Fore, Style
 
 from conans.client.userio import color_enabled
 from conans.util.env import get_env
 
+LEVEL_QUIET = 80  # -q
+
+LEVEL_ERROR = 70  # Errors
+LEVEL_WARNING = 60  # Warnings
+LEVEL_NOTICE = 50  # Important messages to attract user attention.
+LEVEL_STATUS = 40  # Default - The main interesting messages that users might be interested in.
+LEVEL_VERBOSE = 30  # -v  Detailed informational messages.
+LEVEL_DEBUG = 20  # -vv Closely related to internal implementation details
+LEVEL_TRACE = 10  # -vvv Fine-grained messages with very low-level implementation details
+
+# Singleton
+conan_log_level = LEVEL_STATUS
```

Comment:
I think there might be issues in testing as far as I observe the CI

Reply:
Oh, yes, indeed, parallel tests will be an issue (testing was always a big issue with singletons...)
Not sure about the approach, as having to create ConanOutput objects from a ``ConanApp`` instance seems inconvenient, we might need to propagate the ``ConanApp`` or ``ConanOutput`` objects in many more places...

Reply:
Removing the CMake stuff looks like tests are passing again. There were errors with the CMake arguments, but I couldn't explain what happened with others and either, why they don't fail now... I've reviewed how bad would be to avoid the singleton indeed looks pretty bad.